enterprise/management/commands/email_drip_for_missing_dsc_records.py

```diff
@@ -21,6 +28,49 @@ class Command(BaseCommand):
     Django management command for sending an email to learners with missing DataSharingConsent records
     """
 
+    def _get_dsc_url(self, user, course_id, enterprise_customer):
+        """
+        Build the data sharing consent page url
+
+        Arguments
+            user (Object): The user object
+            course_id (String): The course identifier
+            enterprise_customer(Object): Enterprise customer object
+
+        Returns:
+            dsc_url (String): The DSC url
+        """
+        learner_portal_base_url = get_configuration_value(
+            'ENTERPRISE_LEARNER_PORTAL_BASE_URL',
+            settings.ENTERPRISE_LEARNER_PORTAL_BASE_URL
+        )
+        next_url = learner_portal_base_url + '/' + str(enterprise_customer.slug)
+        course_start = None
+        course_details = CourseCatalogApiClient(
```

Comment:
Might be good to add a log message in case if `CourseCatalogApiClient` doesn't return course details

Reply:
Fixed

Reply:
Done